src/http-data-source.ts

```diff
@@ -193,6 +205,7 @@ export abstract class HTTPDataSource<TContext = any> extends DataSource {
       headers: {},
       query: {},
       body: null,
+      memoize: true,
```

Comment:
Could you add jsdoc for this method and document that GET is memorized by default?